### DISCUSSION TITLE
Missing LinuxWelt digital XXL - Das E-Book 1/23 and 2/23

### DISCUSSION THREAD

**Discussion by Geduld2020** [LinuxWelt/LinuxWelt#2] [General]
Sorry, but I'm **missing** the right "LinuxWelt digital XXL" in the ISO-image files concerning 01/23 and 02/23.

The only one content of the pdf-file in **_2023_02_LW_XXL.pdf_** is following text:

"Das Ebook zur LinuxWelt 2022-02 ist im vorliegenden Download-Image nicht enthalten."

**Comment by LinuxWelt** [comment]
Sorry, it's intended. 
The PDF is only available on the physical cover DVD on the LinuxWelt magazine.

**Reply by Geduld2020** [reply]
Many tnx for reply!

But that's not fine. I'm a paying subscriber (PC-WELT Plus Digital Abonnement) and now you reduce scope of services since two magazines on your side without any information to the subscribers. You should not unilaterally restrict the service of our contract! Is there any hope that the ebook will be available online (for PC-WELT Plus Digital users) again?
Please tell me what is the real reason of this. There is enough space on the dvd isn't it? Should I reconsider continuing my subscription?

**Reply by LinuxWelt** [reply]
Hi Geduld2020,
this Github repository is not related to PC-WELT or any subscription. It's public. I am not employed by PC-WELT nor do I speak for them. I publish the images and some systems I develop as free and open-source software for LinuxWelt as an extra service on these Github pages. Would it help you to get you in touch with the publisher to sort out the problems with your subscription?
